src/snowflake/cli/api/config.py

```diff
@@ -47,6 +48,50 @@ class Empty:
 )
 
 
+@dataclass
+class ConnectionConfig:
+    account: Optional[str] = None
+    user: Optional[str] = None
+    password: Optional[str] = None
```

Comment:
This field could be removed from representation `field(default=None, repr=False)`

Reply:
Done